examples/complete/variables.tf

```diff
@@ -27,3 +27,9 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "agent_version" {
+  type        = string
+  description = "The schematics agent version."
+  default     = "1.0.1-beta"
```

Comment:
By adding a default value here, its just one more place we need to maintain it. I guess we need to decide how we want to set the module up in terms of supporting versions, and if we do that we would want to pass the version in the tests. But lets first decide how we will handle versions as per my other comment